pkg/httpserver/httpserver.go

```diff
@@ -50,6 +50,12 @@ func (s *HTTPServer) AddHandler(url string, handler http.Handler) {
 	s.httpServeMux.Handle(url, handler)
 }
 
+// AddHandlerFunc adds an HTTP handler func for the given path on the HTTPServer
+// For changes to be effective, server requires restart
+func (s *HTTPServer) AddHandlerFunc(url string, handler http.HandlerFunc) {
```

Comment:
I don't see this function used in this PR

Reply:
+1, remove if unused.

Reply:
done!